scss/_patterns_subnav.scss

```diff
@@ -0,0 +1,55 @@
+@import 'settings';
+
+@mixin vf-p-subnav {
+  .p-subnav {
+    @extend %vf-is-bordered;
+    @extend %vf-has-box-shadow;
+    @extend %vf-has-round-corners;
+    margin: 0;
+    padding: 0;
+    z-index: 5;
+
+    @media (min-width: $breakpoint-navigation-threshold + 1) {
+      clip: rect(0, 1000px, 1000px, -10px); /* Clip the top of the box-shadow off */
+      margin-top: -1px;
```

Comment:
If using positioning please use `top: -1px` instead of margin.

Reply:
Ah, yes, that's a throwback to a previous styling approach. I'll unify all of that under the position.